trollmoves/tests/test_ssh_server.py

```diff
@@ -149,7 +153,30 @@ def test_scp_open_connection_ssh_exception(self, mock_sshclient_connect):
         with pytest.raises(IOError, match='Failed to ssh connect after 3 attempts'):
             scp_mover.open_connection()
 
-    @patch('paramiko.SSHClient.connect', autospec=True)
+    @patch('trollmoves.movers.SSHClient.connect', autospec=True)
+    def test_scp_open_connection_socket_timeout_exception(self, mock_sshclient_connect):
+        """Check scp get_connection failing with socket timeout."""
+        from trollmoves.movers import ScpMover
+        import logging
+        import socket
+        import sys
```

Comment:
These are in the standard library, can the be imported at the top of the file?

Reply:
yepp